### PR TITLE
Benefits of being an organizer: origin stories, credentials

### PR DESCRIPTION
**Pull request by mjang**
@rosewms @swapnilogale @barbaricyawps

This is just an idea -- but it resembles what I had in mind, based on our discussions during the conference.

(If this idea goes nowhere, I will _not_ feel hurt.)

### CODE REVIEW

**Comment by plaindocs** [comment]
Hey folks, where are we with this? Is it worth dusting off and merging in ?

File: docs/organizer-guide/meetups/meetup-benefits-origin-story.rst [hunk 1]
```diff
@@ -0,0 +1,17 @@
+Benefits of Organizing a Meetup
+===============================
+
+Write the Docs community members have enhanced their careers by organizing a meetup. Beyond leadership, meetup organizers have built skills that have enhanced their careers as Technical Writers, Community Organizers, Developer Relations Engineers, and more.
```
Comment (rosewms) [inline docs/organizer-guide/meetups/meetup-benefits-origin-story.rst:4]: I might go in and wrap the text if we have standards around that? Which I honestly do not know if we do?

File: docs/organizer-guide/meetups/meetup-benefits-origin-story.rst [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+Mike Jang
+---------
+
+Being a meetup leader has enhanced my career. It's made me a better speaker. It's helped me find resources in the open source community. It's opened career options such as Developer Relations. Besides, it's turned me into a community organizer, like the 44th US President! 
+
+I became a Write the Docs meetup organizer after the 2014 conference. I was the only writer in my office, and I needed to connect with people with similar experiences. So before the end of the 2014 conference, I announced that I was starting a Write the Docs meetup in Portland.
+
+Three weeks later, nearly 100 people had signed up. I told a co-worker about this over lunch, and he asked me, "So Mike, when's your first meetup?" That's when our Portland meetup really started!  
+
+<!-- Maybe we now individually add what we included in our LinkedIns for our Meetup experiences -->
```
Comment (rosewms) [inline docs/organizer-guide/meetups/meetup-benefits-origin-story.rst:17]: Remove this line and we can merge what we have in the PR and add to it later

**Comment by rosewms** [comment]
@plaindocs if we remove the last line then it should be good to go, though it is indicating that the page is not in the TOC.

**Comment by plaindocs** [comment]
@rosewms I reckon we need a few more organizer stories before merging? 

As for where to put it, somewhere in here? 
https://github.com/writethedocs/www/blob/main/docs/organizer-guide/index.rst

File: docs/organizer-guide/meetups/meetup-benefits-origin-story.rst (hunk 2 above)
Comment (plaindocs) [inline docs/organizer-guide/meetups/meetup-benefits-origin-story.rst:17]:
```suggestion
```

**Comment by plaindocs** [comment]
Gonna close this for now, we can reopen if anyone is enthusiastic.

**Pull request closed without merging**